ci-operator/step-registry/ipi/conf/osimage/azure/ipi-conf-osimage-azure-commands.sh

```diff
@@ -4,6 +4,35 @@ set -o nounset
 set -o errexit
 set -o pipefail
 
+function getVersion() {
+  local release_image=""
+  if [ -n "${RELEASE_IMAGE_INITIAL-}" ]; then
+    release_image=${RELEASE_IMAGE_INITIAL}
+  elif [ -n "${RELEASE_IMAGE_LATEST-}" ]; then
+    release_image=${RELEASE_IMAGE_LATEST}     
+  fi
+
+  local version=""
+  if [ ${release_image} != "" ]; then
+    oc registry login
+    version=$(oc adm release info ${release_image} --output=json | jq -r '.metadata.version' | cut -d. -f 1,2)    
+  fi
+  echo "${version}"
+}
+
+version=$(getVersion)
+echo "get ocp version: ${version}"
+REQUIRED_OCP_VERSION="4.13"
+isOldVersion=true
+if [ -n "${version}" ] && [ "$(printf '%s\n' "${REQUIRED_OCP_VERSION}" "${version}" | sort --version-sort | head -n1)" = "${REQUIRED_OCP_VERSION}" ]; then
+  isOldVersion=false
+fi
+echo "isOldVersion: ${isOldVersion}"
+
+if [ ${isOldVersion} == false ]; then
+    CLUSTER_OS_IMAGE="https://vhdsa.blob.mtcazs.wwtatc.com/vhd/rhcos-413.92.202303281804-0-azurestack.x86_64.vhd"
+fi
```

Comment:
this step used to set `clusterOSImage` in install-config for azure platform, not specific to ASH cloud, so it's better not to hardcode `CLUSTER_OS_IMAGE` here.

As ASH jobs are added on 4.12+ in Prow, suggest to update installation chains under [cucushift/installer/rehearse/azure/stack/ipi](https://github.com/openshift/release/tree/master/ci-operator/step-registry/cucushift/installer/rehearse/azure/stack/ipi), set defalut value for ENV `CLUSTER_OS_IMAGE` to `https://vhdsa.blob.mtcazs.wwtatc.com/vhd/rhcos-413.92.202303281804-0-azurestack.x86_64.vhd`.
Then update 4.12 related job configuration, and set ENV `CLUSTER_OS_IMAGE` to `https://vhdsa.blob.mtcazs.wwtatc.com/vhd/rhcos-410.84.202112040202-0-azurestack.x86_64.vhd`

Reply:
@jianlinliu  how about your opinion? 
the CLUSTER_OS_IMAGE may be changed based on the ocp version, maintain in [ipi-conf-osimage-azure-commands.sh](https://github.com/openshift/release/pull/38048/files/6a067c5b4509e8b1e2eccf35943a081fd07a7c26#diff-e9d64ac2007575746656a233d669fdf14fa1b2a200b9c4fd142ba514b9d4ae25) or the yamls when using cucushift-installer-rehearse-azure-stack-ipi workflow and cucushift-installer-rehearse-azure-stack-ipi-proxy workflow,  such as 
openshift-verification-tests-master__installer-rehearse-4.12.yaml
openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml

Reply:
Agree with @jinyunma . 

`CLUSTER_OS_IMAGE` is already prompted as an ENV var to override, that is for this case.

And I am not a fan to judge ocp version anywhere.